Resources/views/Core/tab_menu_template.html.twig

```diff
@@ -115,4 +115,11 @@
     </a>
 {% endblock %}
 
-{% block label %}{{ item.label|trans }}{% endblock %}
+{% block label %}
+{{-
+    item.label|trans(
+        item.getExtra('translation_params', {}),
+        item.getExtra('translation_domain', item.getParent('translation_domain'))
```

Comment:
The interface says so : 

```php
    /**
     * @return ItemInterface|null
     */
    public function getParent();
```

and the implementation is simple : 

```php
    public function getParent()
    {
        return $this->parent;
    }
```

Is this ok for you?